mkdocs.yml

```diff
@@ -3,7 +3,9 @@ theme:
   name: material
   palette:
     primary: green
-use_directory_urls: false
```

Comment:
Effect of this edit: when we click the link for the welcome page or the icon at the top left, the browser will go to `regenerativeag.github.io` instead of `regenerativeag.github.io/index.html`